src/lib/services/index.ts

```diff
@@ -30,7 +30,15 @@ export const createServices = (
 ): IUnleashServices => {
     const accessService = new AccessService(stores, config);
     const apiTokenService = new ApiTokenService(stores, config);
-    const clientMetricsService = new ClientMetricsService(stores, config);
+    const clientMetricsService = new ClientMetricsService(
+        stores.clientMetricsStore,
+        stores.strategyStore,
+        stores.featureToggleStore,
+        stores.clientApplicationsStore,
+        stores.clientInstanceStore,
+        stores.eventStore,
+        config.getLogger,
+    );
```

Comment:
To my previous comment, this would the be `new ClientMetricsService(stores, config)`